civihr_employee_portal/views/includes/views_between_operation_dates_filter_handler.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * A filter that allows the between operator to compare dates in 'Y-m-d' format
+ * rather than the default timestamp comparison since absence dates are stored
+ * in 'Y-m-d' format in the db.
+ */
+
+class views_between_operation_dates_filter_handler extends views_handler_filter_date {
+  function op_between($field) {
+    // Use the substitutions to ensure a consistent timestamp.
+    $query_substitutions = views_views_query_substitutions($this->view);
+    $a = intval(strtotime($this->value['min'], $query_substitutions['***CURRENT_TIME***']));
+    $b = intval(strtotime($this->value['max'], $query_substitutions['***CURRENT_TIME***']));
+
+    $a = date('Y-m-d', $a);
+    $b = date('Y-m-d', $b);
+
+    // This is safe because we are manually scrubbing the values.
```

Comment:
Can you explain this comment a bit more please?

Reply:
Copied the comment from the base class.

Reply:
I'm not sure it makes sense to leave the comment as it is then, even if it is copy and paste. If it doesn't make sense can you remove it, or maybe it does make sense and I just don't understand it.